docs/en/commands/element/actions/click.md

```diff
@@ -97,7 +97,10 @@ Not all drivers automatically scroll the element into view and may need to be sc
 
 ### URL Parameters
 
-None
+|name|description|
```

Comment:
These (and the remaining) seem dubious.

Reply:
Yeah, that was a mistake in the template. This will be corrected in future builds.

I think for now, if there's any issues with markdown, we can just ignore them and then we'll do a future build and PR that will build all of the MD files.